adapter/service/chargepoint/tasks.go

```diff
@@ -25,6 +25,7 @@ func handleReporting(serviceRegistry adapter.ServiceRegistry) func() {
 				continue
 			}
 
+<<<<<<< HEAD
```

Comment:
You haven't finished resolving the conflicts, as the conflicts markers are still here.